test/test_DFT_driver.py

```diff
@@ -0,0 +1,369 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+import pytest
+import time
+
+
+from maxwell.driver import *
+from maxwell.dg.mesh1d import *
+from maxwell.dg.dg1d import *
+from maxwell.fd.fd1d import *
+
+
+from nodepy import runge_kutta_method as rk
+
+def dft(x,freq,time):
+    X=[]
+    for f in range(len(freq)):
+        summatory=0.
+        for t in range(len(time)):
+            summatory=summatory + x[t] * np.exp(-2j*np.pi*freq[f]*time[t])
+        X.append(summatory)
+    return X
+
+# def test_DFT_driver():
```

Comment:
It is better to delete these to avoid zombie code. You can always go back with version control.